ambari/src/test/java/io/brooklyn/ambari/AmbariBlueprintLiveTest.java

```diff
@@ -76,47 +76,44 @@ public void tearDown() throws Exception {
 
         return new Object[][]{
                 new Object[]{
-                        "ambari-cluster.yaml", "softlayer", "ams01", "UbuntuOnSoftlayer", MutableMap.<String, String>builder()
-                        .put("minRam", "16384")
-                        .put("minCores", "4")
-                        .put("osFamily", "ubuntu")
-                        .put("osVersionRegex", "12.*")
+                        "ambari-cluster.yaml", "aws-ec2", "eu-central-1", "UbuntuOnAWS", MutableMap.<String, String>builder()
+                        .put("hardwareId", "m4.large")
+                        .put("imageId", "eu-central-1/ami-142ded7b")
                         .put("stopIptables", "true")
                         .build()
                 },
                 new Object[]{
-                        "ambari-cluster.yaml", "softlayer", "ams03", "CentOSOnSoftlayer", MutableMap.<String, String>builder()
-                        .put("minRam", "16384")
-                        .put("minCores", "4")
-                        .put("osFamily", "centos")
-                        .put("osVersionRegex", "6.*")
+                        "ambari-cluster.yaml", "aws-ec2", "eu-central-1", "CentOSOnAWS", MutableMap.<String, String>builder()
+                        .put("hardwareId", "m4.large")
+                        .put("imageId", "eu-central-1/ami-14211709")
                         .put("stopIptables", "true")
                         .build()
                 },
                 new Object[]{
-                        "ambari-cluster.yaml", "softlayer", "lon02", "RHEDOnSoftlayer", MutableMap.<String, String>builder()
-                        .put("minRam", "16384")
-                        .put("minCores", "4")
-                        .put("osFamily", "rhel")
-                        .put("osVersionRegex", "6.*")
+                        "ambari-cluster.yaml", "aws-ec2", "eu-central-1", "RHELOnAWS", MutableMap.<String, String>builder()
+                        .put("hardwareId", "m4.large")
+                        .put("imageId", "eu-central-1/ami-cd362ca1")
```

Comment:
Thanks @duncangrant, @bostko  
I will leave it with the image Id.
I did it that way, because Brooklyn still fails to find the correct images when regex is used in some cases. I will follow up on the issue which is handled in https://issues.apache.org/jira/browse/BROOKLYN-394